lib/taiko.js

```diff
@@ -1868,15 +1868,23 @@ function dropDown(attrValuePairs, ...args) {
                     this.selectedIndex = value.index;
                     found_value = true;
                     let event = new Event('change');
-                    this.dispatchEvent(event);
+                    try{
+                        this.dispatchEvent(event);
+                    }catch(e){
+                        return 'Error occured while dispatching event ' + e;
```

Comment:
This error message is not user friendly. Plus it gives a dump of the stack.
Please handle this is a way that a user can understand and take action. 

Also do not throw errors as a string if you want to identify this throw a generic appError
https://www.npmjs.com/package/apperror